utils/abrt-action-analyze-java.c

```diff
@@ -132,23 +132,23 @@ backtrace_from_file(const char *file_name)
 static char *
 work_out_list_of_remote_urls(struct sr_java_stacktrace *stacktrace)
 {
-    struct strbuf *remote_files_csv = libreport_strbuf_new();
+    g_autoptr(GString) remote_files_csv = g_string_new(NULL);
     struct sr_java_thread *thread = stacktrace->threads;
     while (NULL != thread)
     {
         struct sr_java_frame *frame = thread->frames;
         while (NULL != frame)
         {
-            if (NULL != frame->class_path && libreport_prefixcmp(frame->class_path, "file://") != 0)
+            if (NULL != frame->class_path && g_str_has_prefix(frame->class_path, "file://") == 0)
             {
                 struct stat buf;
                 if (stat(frame->class_path, &buf) && errno == ENOENT)
                 {
-                    if (strstr(remote_files_csv->buf, frame->class_path) == NULL)
+                    if (strstr(remote_files_csv->str, frame->class_path) == NULL)
                     {
                         log_debug("Adding a new path to the list of remote paths: '%s'", frame->class_path);
-                        libreport_strbuf_append_strf(remote_files_csv, "%s%s",
-                                remote_files_csv->buf[0] != '\0' ? ", " : "",
+                        g_string_append_printf(remote_files_csv, "%s%s",
+                                remote_files_csv->str[0] != '\0' ? ", " : "",
```

Comment:
Same remark with `len`. If you stick to checking for null byte, can you please switch the logic so that it's `x == y ? if_eq : if_not_eq`, rather than `x != ? if_not_eq : if_eq`?